spec/truffle/tags/core/kernel/spawn_tags.txt

```diff
@@ -59,3 +59,31 @@ slow:Kernel#spawn with Integer option keys maps the key to a file descriptor in
 ci:Kernel#spawn unsets other environment variables when given a true :unsetenv_others option
 ci:Kernel#spawn unsets other environment variables when given a non-false :unsetenv_others option
 ci:Kernel#spawn does not unset environment variables included in the environment hash
+slow:Kernel#spawn is a private method
+slow:Kernel#spawn executes the given command
+slow:Kernel#spawn raises an ArgumentError if given a negative :pgroup option
+slow:Kernel#spawn raises a TypeError if given a symbol as :pgroup option
+slow:Kernel#spawn raises an ArgumentError if passed no command arguments
+slow:Kernel#spawn raises an ArgumentError if passed env or options but no command arguments
+slow:Kernel#spawn raises an ArgumentError if passed env and options but no command arguments
+slow:Kernel#spawn raises an Errno::ENOENT for an empty string
+slow:Kernel#spawn raises an Errno::ENOENT if the command does not exist
+slow:Kernel#spawn raises an ArgumentError when passed a string key in options
+slow:Kernel#spawn raises an ArgumentError when passed an unknown option key
+slow:Kernel#spawn with a single argument subjects the specified command to shell expansion
+slow:Kernel#spawn with a single argument creates an argument array with shell parsing semantics for whitespace
+slow:Kernel#spawn with a single argument calls #to_str to convert the argument to a String
+slow:Kernel#spawn with a single argument raises an ArgumentError if the command includes a null byte
+slow:Kernel#spawn with a single argument raises a TypeError if the argument does not respond to #to_str
+slow:Kernel#spawn with multiple arguments does not subject the arguments to shell expansion
+slow:Kernel#spawn with multiple arguments preserves whitespace in passed arguments
+slow:Kernel#spawn with multiple arguments calls #to_str to convert the arguments to Strings
+slow:Kernel#spawn with multiple arguments raises a TypeError if an argument does not respond to #to_str
+slow:Kernel#spawn with a command array uses the first element as the command name and the second as the argv[0] value
+slow:Kernel#spawn with a command array does not subject the arguments to shell expansion
+slow:Kernel#spawn with a command array preserves whitespace in passed arguments
+slow:Kernel#spawn with a command array calls #to_ary to convert the argument to an Array
+slow:Kernel#spawn with a command array calls #to_str to convert the first element to a String
+slow:Kernel#spawn with a command array calls #to_str to convert the second element to a String
+slow:Kernel#spawn with a command array raises an ArgumentError if the Array does not have exactly two elements
+slow:Kernel#spawn with a command array raises a TypeError if an element in the Array does not respond to #to_str
```

Comment:
I guess it's due to me changing some specs in 988b8e8fd53c1ba76df9f5235da4dcb45a1930f4.
Anyway it's fine, it's the automatic tagging of slow specs.